### PR TITLE
kubernetes: More post merge fixes

### PR DESCRIPTION
**Pull request by petervo**
Firefox doesn't trigger the click event when it is delayed.

### CODE REVIEW

**Comment by stefwalter** [comment]
```
pkg/kubernetes/scripts/topology.js:138:71: Missing semicolon.
```

File: pkg/kubernetes/views/deploy.html [hunk 1]
```diff
@@ -28,14 +28,14 @@ <h3 class="modal-title" translatable="yes">Deploy Application</h3>
                 </td>
                 <td class="dialog-wrapper">
                     <div ng-if="selected.type == 'manifest'">
+                        <input id="deploy-app-manifest-file" type="file"
+                            onchange="angular.element(this).scope().fileSelected(this)">
                         <button id="deploy-app-manifest-file-button" class="btn btn-default form-control" type="button"
                             ng-class="{manifest_file_default: !fields.filename, manifest_file: fields.filename}"
-                            ng-click="fileBtnClick($event)">
+                            onclick="document.getElementById('deploy-app-manifest-file').click()">
```
Comment (stefwalter) [inline pkg/kubernetes/views/deploy.html]: This won't or shouldn't work with strict Content-Security-Policy.

Comment (petervo) [inline pkg/kubernetes/views/deploy.html]: It seems to work here, so I guess the on change above is bad then too?

Comment (petervo) [inline pkg/kubernetes/views/deploy.html]: removed both on attributes and switched to a directive.

File: pkg/kubernetes/views/file-button.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<input id="deploy-app-manifest-file" type="file">
+<button id="deploy-app-manifest-file-button" class="btn btn-default form-control" type="button"
+    ng-class="{manifest_file_default: !fields.filename, manifest_file: fields.filename}">
+    <span ng-if="!fields.filename" translatable="yes">Select Manifest File...</span>
+    <span ng-if="fields.filename" translatable="yes">{{ fields.filename }}</span>
```
Comment (stefwalter) [inline pkg/kubernetes/views/file-button.html]: Doesn't need translatable.

**Pull request closed without merging**